src/apps/backend/settings/development.py

```diff
@@ -6,10 +6,12 @@ class DevelopmentSettings:
     LOGGER_TRANSPORTS: tuple[str] = ("console",)
     MONGODB_URI: str = "mongodb://localhost:27017/frm-boilerplate-dev"
     SMS_ENABLED: bool = False
+    USE_PROXY: bool = False # Set as True if server is running behind proxy
```

Comment:
Remove comment, rename variable name to `IS_SERVER_RUNNING_BEHIND_PROXY`. Code should itself be readable, and comments like this here (and below) should be avoided.